plyer/facades/maps.py

```diff
@@ -0,0 +1,83 @@
+'''
+Maps
+=======
+The :class:`Maps` creates a client for accessing the default Maps API.
+
+Holds features such as opening a location by
+address & latitude/longitude, create queries, or find directions between
+two points
+
+Simple Examples
+---------------
+
+Perform a search::
```

Comment:
Maybe we should specify that the search results are not returned, but instead the OS default map app gets opened, and will show the search results?